Signal/src/ViewControllers/ConversationView/MessagesViewController.m

```diff
@@ -1531,6 +1531,24 @@ - (BOOL)collectionView:(JSQMessagesCollectionView *)collectionView
     return YES;
 }
 
+- (BOOL)collectionView:(UICollectionView *)collectionView
```

Comment:
This was just a straight move - colocated with the other CollectionViewDelegate methods.